static/app/views/dashboardsV2/index.tsx

```diff
@@ -10,6 +10,7 @@ import Detail from './detail';
 type Props = RouteComponentProps<{orgId: string; dashboardId: string}, {}> & {
   organization: Organization;
   children: React.ReactNode;
+  location: Location;
```

Comment:
This implicit `Location` type is different from what is provided by the `history` package. You'd need to explicitly import it:

```
import {Location} from 'history';
```